magstock/templates/regextra.html

```diff
@@ -1,17 +1,24 @@
 
 <script type="text/javascript">
     if (window.BADGE_TYPES) {
-        BADGE_TYPES.options[0].description = 'Includes three day admission and campground fees';
-        if (BADGE_TYPES.options[2]) {
-            BADGE_TYPES.options[2].description = 'T-shirt level plus additional swag. (Available until {{ c.SUPPORTER_DEADLINE|datetime:'%b %d' }}. Limited quantities.)';
-        }
-        {% if c.SHIRT_LEVEL in c.PREREG_DONATION_TIERS %}
-            BADGE_TYPES.options[1] = {
-                title: 'Add a T-Shirt',
-                description: 'Attendee level plus a MAGStock-themed t-shirt',
-                extra: {{ c.SHIRT_LEVEL }}
-            };
-        {% endif %}
+        $.each(BADGE_TYPES.options, function (i, badgeType) {
+            switch (BADGE_TYPES.options[i].extra) {
```

Comment:
+1 for a second time on this whole PR.

Though as an even-nitpickier-than-before style suggestion for next time, you could just say ``switch(badgeType.extra)`` here.